types/signatures.go

```diff
@@ -57,7 +57,7 @@ type (
 		SiafundOutputs           []uint64
 		MinerFees                []uint64
 		ArbitraryData            []uint64
-		Signatures               []uint64
+		TransactionSignatures    []uint64
```

Comment:
what exactly is unambiguous about `txn.Signatures`? "transaction dot transaction signatures" stutters.